src/main/java/com/robomwm/prettysimpleshop/feature/ShowoffItem.java

```diff
@@ -157,7 +157,7 @@ private void onChunkUnload(ChunkUnloadEvent event)
         while (locations.hasNext()) //can optimize later via mapping chunks if needed
         {
             Location location = locations.next();
-            if (location.getChunk() == event.getChunk())
+            if (location.getBlockX() >> 4 == event.getChunk().getX() && location.getBlockZ() >> 4 == event.getChunk().getZ())
```

Comment:
What's up here? Is it possible for there to be two different Chunks to exist on `getChunk` which correspond to the same world/chunk coordinate?

Reply:
might just be leftover, was just making sure